content/en/docs/tutorials/hello-minikube.md

```diff
@@ -142,7 +142,7 @@ recommended way to manage the creation and scaling of Pods.
     kubectl config view
     ```
 
-1. View application logs for a container in a pod.
+1. View application logs for a container in a pod. Note, the name passed here should be the one shown in the output from the `kubectl get pods` command.
```

Comment:
```suggestion
1. View application logs for a container in a pod. Make sure to replace `hello-node-5f76cf6ccf-br9b5` with the name of the pod from `kubectl get pods` command output.
```

Reply:
If you are disagree the suggestions, you cannot mark it as resolved.

Reply:
> I'm happy with that

That is not disagreeing, that is saying that I'm happy with it.

Reply:
> > I'm happy with that
> 
> That is not disagreeing, that is saying that I'm happy with it.

@digininja,
You need to push the updated branch with above suggested changes, then only we can mark it as resolved.
Please work on it and make the changes available in PR.